docs/_documentations/codewind-eclipse-quick-guide.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: docs
+title: Codewind in Eclipse
+description: Take advantage of Codewind's tools to help build high quality cloud native applications regardless of which IDE or language you use.
+keywords: Codewind, VS Code, microservice
+duration: 5 minutes
+permalink: codewind-eclipse-quick-guide
+---
+
+# Getting Started with Codewind in Eclipse 
+
+## Objectives
+* Develop a simple microservice, using Eclipse Codewind on Eclipse
+* Install VS Code and Codewind 
+
+## Overview 
+Eclipse Codewind provides the ability to create application projects from these `Application Stacks` that your company has built, enabling developers to focus on their code and not infrastructure and Kubernetes.  Application deployments to Kubernetes occur via pipelines when developers commit their local code to the correct Git repos Kabanero is managing via webhooks.    
+
+Eclipse Codewind provides the ability to create projects based on a variety of different template types.  These include IBM Cloud starters, OpenShift Do (odo), and Appsody templates. Today, there are templates for: IBM Cloud Starters, odo, Eclipse MicroProfile/Java EE, Springboot, Node.js, Node.js with Express, Node.js with Loopback.
+
```

Comment:
Last item in paragraph needs an `and` before it so `Node.js with Express, and Node.js with Loopback.`